packages/product-components/src/components/AssetLogo/AssetLogo.stories.tsx

```diff
@@ -24,15 +24,26 @@ export const AssetLogo: StoryObj<AssetLogoProps> = {
         contractAddress: '0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48',
         shouldTryToFetch: true,
         placeholder: 'USDC',
+        showNetworkIcon: false,
         ...getFramePropsStory(allowedAssetLogoFrameProps).args,
     },
     argTypes: {
+        ...getFramePropsStory(allowedAssetLogoFrameProps).argTypes,
```

Comment:
nitpick: I would rather keep frame props at the end, we use it this way in all components

Reply:
I put it here to prevent overriding those props below..